### PR TITLE
Sprint1 code review stegemtc

### PR DESCRIPTION
**Pull request by trey368s**
Pull request for code review changes

### CODE REVIEW

File: src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java [hunk 1]
```diff
@@ -26,6 +26,9 @@ public String addFacilityPage() {
         return "AddFacility";
     }
 
+    @RequestMapping("/index.html")
+    public String returnIndex() {return "index";}
+
```
Comment (trey368s) [inline src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java:31]: Search function leads to index.html which has a Whitepage error.

File: src/main/resources/templates/index.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
     <ul class="nav">
         <li><a href="index.html">Search</a></li>
-        <li><a href="AddFacility.html">Add Faculty</a></li>
+        <li><a href="addfacility.html">Add Faculty</a></li>
     </ul>
```
Comment (trey368s) [inline src/main/resources/templates/index.html:17]: The name AddFacility.html is inconstant among files leading to a whitepage error

File: src/main/resources/templates/style.css [hunk 3]
```diff
@@ -101,10 +98,8 @@ input[type=submit]:hover {
     width: 350px;
     height: 300px;
     border: 1px solid black;
-    padding 50px;
```
Comment (trey368s) [inline src/main/resources/templates/style.css:104]: CSS attribute is missing colon.

**Comment by trey368s** [comment]
**Analysis of the program**
I think that this is a very well though out program it is much further along in development than ours and it helps me see where we should be taking our project. I can see that the finished product is going to work well because of the planning and they should have no problem getting a good grade and completed finished product. 
**Was the program available in UC GitHub on time?**
Yes the program was available in GitHub on time. 
**Is the program easy for you, as an outsider, to understand?**
Yes the naming conventions and documentation make the project fairly easy to understand. 
**Does the program compile?**
The program compiles but it has a problem loading the CSS on my computer, though I suspect it is my Antivirus because I can get the CSS to load in the IntelliJ webserver.
**Links to three commits you made to your group's repository during Sprint 1.**
https://github.com/trey368s/Market_Data_API/commit/c2f2f92aeb8426e79ffcc4bcedef13a81830e98f
https://github.com/trey368s/Market_Data_API/commit/10686000ee89f0b509c6a6a27ebed6487169aa2e
https://github.com/trey368s/Market_Data_API/commit/2f28c463789c8d8b24eee16afc7543f079cf6cd6

File: src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java [hunk 4]
```diff
@@ -21,11 +21,14 @@ public String index() {
         return "index";
     }
 
-    @RequestMapping("/AddFacility.html")
+    @RequestMapping("/addfacility.html")
     public String addFacilityPage() {
-        return "AddFacility";
+        return "addfacility";
```
Comment (discospiff) [inline src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java:26]: Lowercase is best... however, this did appear in a previous review.  When that occurs, first in wins.

File: src/main/resources/templates/index.html [hunk 5]
```diff
@@ -2,7 +2,7 @@
 <html lang="en" xmlns:th="http://www.thymeleaf.org">
 <head>
     <meta charset="UTF-8">
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" type="text/css" Href="style.css">
```
Comment (discospiff) [inline src/main/resources/templates/index.html:5]: Should be lowercase href.

File: src/main/resources/templates/style.css [hunk 6]
```diff
@@ -65,8 +63,7 @@ text-decoration: none;
     height: 200px;
     border: 1px solid black;
     padding: 50px;
-    margin: auto;
-    margin-top: 25px;
+    margin: 25px auto auto;
```
Comment (discospiff) [inline src/main/resources/templates/style.css:66]: This appeared in a previous review.  First in wins.

**Comment by discospiff** [comment]
Several suggestions here appeared in a previous code review.  See my comments on those branches.

**Pull request closed without merging**